inc/class_core.php

```diff
@@ -519,6 +521,8 @@ public function get_asset_url($path = '', $use_cdn = true)
 			$url = $path;
 		}
 
+		$plugins->run_hooks('get_asset_url_end', $url);
```

Comment:
And how would you suggest implementing this then? `run_hooks` is only slow if there are hooks associated, and if there are hooks associated then obviously the hook is useful...